.github/workflows/nodejs.yml

```
@@ -8,6 +8,7 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
```

If you're editing this file anyways...
```suggestion
  pull_request:
```
This way, tests are run even on PRs that are children of other branches, such as branches you're merging into a feature branch.